### PR TITLE
VE-2061: Add basic rules to determine when to show EBS

### PR DESCRIPTION
**Pull request by pauloslund**

### CODE REVIEW

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php [hunk 1]
```diff
@@ -19,8 +24,21 @@ public static function saveUserResponse( $val ) {
 			$wgUser->setOption( $option_name, $timestamp );
 		}
 	}
+	// PROTECTED
+
+	protected function isCorrectPage() {
+		$title = $this->getContext()->getTitle();
+		$specialPageName = Transaction::getAttribute(Transaction::PARAM_SPECIAL_PAGE_NAME);
```
Comment (inez) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php]: Would make sense to check of it's "any" special page first? Title::isSpecial

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php [hunk 2]
```diff
@@ -19,8 +24,21 @@ public static function saveUserResponse( $val ) {
...
+
+		return $title->isContentPage() || $specialPageName === 'WikiActivity' || $specialPageName === 'Recentchanges';
+	}
+
+	protected function userIsPowerUser() {
```
Comment (inez) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php]: Two times user in the name of the method - what for?

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php [hunk 3]
```diff
@@ -19,8 +24,21 @@ public static function saveUserResponse( $val ) {
...
+		$user = $this->getContext()->getUser();
+		return $user->isPowerUser();
+	}
 
-	public static function canOpenEBS( ) {
+	protected function canOpenEBS( ) {
```
Comment (inez) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php:41]: I always have problem with this... If something could be static why not to keep it as such - could be used in more places then - specifically I will need it to decide about adding js.

**Comment by inez** [comment]
We need to check for wiki language too

**Comment by pauloslund** [comment]
Can language check be done in another PR?

**Pull request merged**